src/styles/_components/_inputs.scss

```diff
@@ -529,11 +528,52 @@ $checkbox-size: 32px;
         position: absolute;
         left: -99999px;
     }
-    
+
     &__label {
         position: absolute;
         width: 100%;
         height: 100%;
         z-index: 2;
     }
 }
+
+@mixin select() {
+    height: $input-height;
```

Comment:
below this line please add:
` @include height-modifiers();`

This will allow you to have "free" &--small, &--medium and &--large modifiers.